src/_utils.ts

```diff
@@ -158,3 +158,16 @@ export function doesDependencyExist(
     return false;
   }
 }
+
+export function sanitizePackageManagerName(
+  name: string,
+): [any, string | undefined] {
+  const sanitized = name.replace(/^\W+/, "");
+  if (name !== sanitized) {
```

Comment:
Wondering why not simply using a regex test /^\w+$`

Reply:
Ah, at first I thought we would be sanitizing the full `packageManager` string which potentially includes the version, but since the version is already split we could test/exec for just the word characters.

Reply:
you are right i forgot that context.